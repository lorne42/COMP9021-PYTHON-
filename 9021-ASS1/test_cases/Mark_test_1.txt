During my travels, I stumbled upon a gathering of Sirs David, Emily, and Alex.
Sir David, with a smile, declared: "All of us are Knights!" On the other hand, Sir Emily remarked: "At most one of us is a Knight." Sir Alex just watched silently. Who among them speaks the truth?