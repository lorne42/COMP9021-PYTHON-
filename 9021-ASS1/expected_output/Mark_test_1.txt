The Sirs are: Alex David Emily
There is a unique solution:
Sir Alex is a Knave.
Sir David is a Knave.
Sir Emily is a Knight.